lib/openstudio/analysis/translator/workflow.rb

```diff
@@ -65,6 +65,7 @@ def initialize(osa_filename, options = {})
           @osw_version = '0.0.1'
           @options = options
           @file_paths = options[:file_paths] ? options[:file_paths] : []
+          @file_paths << '../lib'
```

Comment:
Looks like this will always be added, no matter what the user passed in the :file_path. Find by me.